src/components/Search/search.jsx

```diff
@@ -7,14 +9,24 @@ import './search.scss';
 
 export const Search = () => {
   const [request, setRequest] = useState('');
+  const history = useHistory();
   const dispatch = useDispatch();
   const sortParam = useSelector(getCurrentSortParam);
   const filterParam = useSelector(getCurrentFilterParam);
+  const { searchRequest } = useRouteMatch().params;
```

Comment:
not an error, but you can do it in this way:
```js
const { params: searchRequest } = useRouteMatch();
```

Reply:
thanks! fixed